Origin/task 11

Shurachok opened the pull request:
task 11 (html + css for mob/tab/desk)

Shurachok committed:
task 11 (html+css mob/tab/desk)

Shurachok committed:
formatting

Yurii-Ilkevych reviewed (commented):
кнопка забронювати столик повинна бути в один рядок на трьох точках перелому також повинна мати фокус також на таблет версії права сторона меню знизу знаходиться занадто високо проти лівої бачу там маджини стоять замалі 8px а на макеті 11xp на мобайл версії також маджини 8 замість 11 на десктоп версії також маджини не в порядку перевірь їх згідно макету будь ласка

Shurachok committed:
css

підправив margins
в кнопці забронювати столик прописано відкриття модального вікна

Shurachok commented:
виправив margins,проте на макеты таблету та десктопу між колонками меню різні відступи: різниця в 5px є

Shurachok committed:
margin additionaly corrected

Yurii-Ilkevych commented on src/css/task-eleven.css:
```diff
@@ -0,0 +1,237 @@
+  letter-spacing: 0.02em;
+  color: var(--akzent-yellow-colo);
+  background-color: black;
+  padding: 14px 23px;
```
забронювати столик має бути в 1 рядок просто поміняй падинг друге значення з 23 px на 22px і стане все ок

Yurii-Ilkevych reviewed (commented):
а також десктоп та таблет нічого не змінилось колона меню з правої сторони візуальна має бути трошкі нижче

Shurachok committed:
margin corrected
